pages/api/posts/publish.js

```diff
@@ -61,13 +66,11 @@ export default async (req, res) => {
     if (session && session.user) {
       const granted = await isAllowedToPost(session.user.nickname);
       if (granted) {
-        const message = "Some commit message";
+        const message = "News publish";
         const author = {
           name: session.user.name,
           email: session.user.email
         };
-        req.body.team = sanitize(req.body.team);
-
         const params = { author, message, data: req.body };
         return Promise.all([
           await publishTeam(params),
```

Comment:
await are not necessary in the promise.all imho

Reply:
Indeed.